packages/core/helper-plugin/src/features/AppInfo.tsx

```diff
@@ -94,45 +82,11 @@ const AppInfoProvider = ({
   return <AppInfoContext.Provider value={contextValue}>{children}</AppInfoContext.Provider>;
 };
 
-AppInfoProvider.defaultProps = {
-  autoReload: undefined,
-  communityEdition: undefined,
-  currentEnvironment: undefined,
-  dependencies: undefined,
-  latestStrapiReleaseTag: undefined,
-  nodeVersion: undefined,
-  projectId: undefined,
-  strapiVersion: undefined,
-  useYarn: undefined,
-  userId: null,
-};
-
-AppInfoProvider.propTypes = {
-  children: PropTypes.node.isRequired,
-  autoReload: PropTypes.bool,
-  communityEdition: PropTypes.bool,
-  currentEnvironment: PropTypes.string,
-  dependencies: PropTypes.object,
-  latestStrapiReleaseTag: PropTypes.string,
-  nodeVersion: PropTypes.string,
-  projectId: PropTypes.string,
-  setUserDisplayName: PropTypes.func.isRequired,
-  shouldUpdateStrapi: PropTypes.bool.isRequired,
-  strapiVersion: PropTypes.string,
-  useYarn: PropTypes.bool,
-  userDisplayName: PropTypes.string.isRequired,
-  userId: PropTypes.string,
-};
-
 /* -------------------------------------------------------------------------------------------------
  * Hook
  * -----------------------------------------------------------------------------------------------*/
 
-/**
- * @preserve
- * @returns {AppInfoContextValue}
- */
-const useAppInfo = () => React.useContext(AppInfoContext);
+const useAppInfo: () => AppInfoContextValue | EmptyObject = () => useContext(AppInfoContext);
```

Comment:
Do we need to type this? Could it not be inferred? 😄

Reply:
True, I guess I was being overzealous. I'll update the PR and remove it.